packages/manager/modules/pci/src/projects/project/serving/namespace/models/add/translations/Messages_en_GB.json

```diff
@@ -13,7 +13,8 @@
   "pci_projects_project_serving_namespace_models_add_instances": "Instance configuration",
   "pci_projects_project_serving_namespace_models_add_instances_info": "During preview only one type of instances is available. Additional flavors will be created to fit specific needs.",
   "pci_projects_project_serving_namespace_models_add_autoscaling": "Auto Scaling",
-  "pci_projects_project_serving_namespace_models_add_storage_path_info": "Select the path to your model directory on the Object Storage",
+  "pci_projects_project_serving_namespace_models_add_storage_path_info_file": "Select the path to your model file on the Object Storage. Your file must ends with: h5 / pmml or onnx",
```

Comment:
As mentioned else where by @FredericEspiau, you don't have to provide english translations, but just french translations, as the french content will be translated before to be prodded, by the translation team.

Reply:
As we are in beta, we heard that we needed to translate ourself ... We may need some discussion about this.